core/src/main/java/org/kiwix/kiwixmobile/core/CoreApp.kt

```diff
@@ -76,6 +78,15 @@ abstract class CoreApp : Application() {
   }
 
   override fun onCreate() {
+    if (MissingSplitsManagerFactory.create(this).disableAppIfMissingRequiredSplits()) {
+      AlertDialog.Builder(this)
+        .setMessage(org.kiwix.kiwixmobile.core.R.string.missing_split_version_for_objectbox)
+        .setPositiveButton(R.string.yes) { dialog, which ->
+          android.os.Process.killProcess(android.os.Process.myPid())
```

Comment:
@MohitMaliFtechiz @kelson42 Is this way to properly handle by giving the message to download from the google play store and kill the application 🤔 ? I'm sorry I can't find any solution in the ticket.

Reply:
hi @gouri-panda ,

if user are using the device architecture which is not supported then i think there is no way he can use the app so there is not point for navigating to playstore.